src/partials/head-meta.hbs

```diff
@@ -61,6 +69,7 @@
     {{/with}}
 
     <meta property="og:url" content="https://neo4j.com{{ page.attributes.canonical-root }}{{ page.url }}">
+    {{#unless (eq "docs" page.attributes.theme)}}
```

Comment:
question: Why you don't want to add the `<link rel="canonical">` tag on the docs?

Reply:
First it was noticed that there were two (because we have an absolute URL), and then that they didn't match.

Also, because the SEO understanding was that if we canonicalize the “current” version google will remove the version numbered page, and if we canonicalize the version number page, google will remove the “current” version. or at least that’s our current thinking.

This is probably not apparent for other content the bundle works with if it is not versioned.

Reply:
That's why we should use `page.canonicalUrl`. It seems that you already have a symbolic name for the most up to date version of a component ("current").
What `page.canonicalUrl` does is the following:

| URL | `page.canonicalUrl` |
| -- | -- |
| https://neo4j.com/docs/getting-started/current/ | https://neo4j.com/docs/getting-started/current/
| https://neo4j.com/docs/getting-started/4.1/ | https://neo4j.com/docs/getting-started/current/ |
| https://neo4j.com/docs/getting-started/4.2/ | https://neo4j.com/docs/getting-started/current/ |


> Also, because the SEO understanding was that if we canonicalize the “current” version google will remove the version numbered page, and if we canonicalize the version number page, google will remove the “current” version.

I think the site will get penalized because the content is almost identical. I'm not a SEO expert but I think that's the idea behind this feature.

So to clarify you don't want to add `<link rel="canonical">` when the content is versioned? I think we should add a comment to document this decision for our future selves :sweat_smile:

Reply:
I think this probably needs to be opened out into wider discussion with @davidoliverSP2 and @danfitzpatrick. I'm not canonical-smart enough to make the call 😄 

I do know that identical-looking content is an SEO negative, and is one of the areas that gets flagged in our SEO reports.